tests/test-data/deterministic/models/infer.wppl

```diff
@@ -0,0 +1,9 @@
+var model = function() {
+  return flip(1);
+};
+[
+  // Test the use of ES6 object literal shorthand.
+  sample(Infer({model})),
+  sample(Infer({model: model})),
+  sample(Infer({model() { return model(); }}))
```

Comment:
I was initially surprised that this works in Node v0.12, given that it doesn't support the ES6 object literal extensions. But Esprima transforms this into the corresponding standard syntax tree, so everything works out. :-)